ISHPermissionKit/ISHPermissionsViewController.m

```diff
@@ -205,6 +205,10 @@ - (void)permissionRequestViewController:(ISHPermissionRequestViewController *)vc
         [self transitionToPermissionCategoryAtIndex:nextIndex];
         [self setCurrentIndex:self.currentIndex + 1];
     }
+
+    if (error != nil && self.errorBlock) {
+        self.errorBlock(vc.permissionCategory, (NSError * _Nonnull)error);
```

Comment:
I don't think we promise a certain order here, but it's probably still better to do this consistently.